qiskit/providers/aer/noise/noise_model.py

```diff
@@ -307,7 +314,11 @@ def from_backend(cls, backend,
             raise NoiseError(
                 "NoiseModel.from_backend does not currently support V2 Backends.")
         if isinstance(backend, (BaseBackend, BackendV1)):
-            properties = backend.properties()
+            # TODO This if state below is not so good.
+            if datetime:
+                properties = backend.properties(datetime=datetime)
```

Comment:
[`BackendV1.properties`](https://github.com/Qiskit/qiskit-terra/blob/77219b5c7b7146b1545c5e5190739b36f4064b2f/qiskit/providers/backend.py#L147) does not take any arguments.

(Though deprecation warning is shown), use of `BackendProperties` instead of `BackendV1` is a work around. Also, defining custom backend that returns specific `BackendProperties` can resolve this issue.

Reply:
Thank you for reviewing it.
I forgot that [BackendV1.properties](https://github.com/Qiskit/qiskit-terra/blob/77219b5c7b7146b1545c5e5190739b36f4064b2f/qiskit/providers/backend.py#L147) does not have that argument.
I understand that by using `BackendProperties` what I want can be done, but I don't think it is a good idea to use a 
deprecated function.

I have added a code to check the class of backend is  `IBMQBackend`, whose class method `properties` has the argument `datetime`.
Could you please review it?

Reply:
`IBMQBackend` is in [qiskit-ibmq-provider](https://github.com/Qiskit/qiskit-ibmq-provider). qiskit-aer can not has dependency on it.
I believe that this issue should be resolved in `qiskit-ibmq-provider` (or user codes) and not in Aer. [`IBMQBackend.properties()`](https://github.com/Qiskit/qiskit-ibmq-provider/blob/master/qiskit/providers/ibmq/ibmqbackend.py#L410-L413) may allow cache not-the-latest properties ([current code](https://github.com/Qiskit/qiskit-ibmq-provider/blob/master/qiskit/providers/ibmq/ibmqbackend.py#L456-L457) does not).

Reply:
Thank you for your comment.

> IBMQBackend is in [qiskit-ibmq-provider](https://github.com/Qiskit/qiskit-ibmq-provider). qiskit-aer can not have dependency on it.
I understand your policy.

> I believe that this issue should be resolved in qiskit-ibmq-provider (or user codes) and not in Aer.
I disagree with that, I think the code to use local simulator should be in this repository, otherwise many similar codes should be included in a lot of repositories.
For example in this noise simulator, each library to control real quantum computers needs similar code to use a local simulator, or users must write each time, which is not good to use and maintain, I think.`

I understand the reason, thanks.